components/definitions/_definitions.scss

```diff
@@ -0,0 +1,195 @@
+.definition {
+  display: none;
+  border-top: 1px solid $color-borders;
+  margin-top: 3em;
+
+  .no-js & {
+    display: block;
+  }
+}
+
+.definition__list {
+  margin: 0;
+  padding: 1.5em 0 1em;
+  position: relative;
+}
+
+.definition__title {
+  display: inline-block;
+}
+
+.definition__backtotop {
+  $size: 1.5rem;
+
+  background: url("#{$static}/img/icons/icons--definition-arrow.svg") no-repeat center center;
+  background-size: auto;
+  text-align: center;
+  width: $size;
+  height: $size;
+  display: inline-block;
+  text-indent: -999999px;
+  overflow: hidden;
+  vertical-align: top;
+  @include mq(1100px) {
+    position: absolute;
+    top: 1.5em;
+    left: -2em;
+  }
+}
+
+.definition__description {
+  margin: 0;
+}
+
+.definition__anchor {
+  border: 0;
+  text-decoration: none;
+  color: $color-links;
+  font-weight: inherit;
+  padding: 0;
+  border-width: 0 0 1px;
+  border-radius: 0;
+  border-style: dashed;
+  margin-right: 0.6em;
+  appearance: none;
+  background-color: transparent;
+  position: relative;
+  &:hover {
+    text-decoration: none;
+    color: $color-links-hover;
+    border-color: $color-links-hover;
+  }
+
+  &:focus {
+    color: #fff;
+    background-color: $color-links;
+    outline: none;
+  }
+
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: -0.6em;
+    background: transparent url("#{$static}/img/icons/icons--definition-anchor.svg") no-repeat 50%;
+    background-size: auto;
+    padding: 0;
+    display: inline-block;
+    white-space: nowrap;
+    line-height: 0;
+    vertical-align: super;
+    margin-left: 0;
+    width: 0.6em;
+    height: 0.6em;
+  }
+}
+
+@keyframes fadein {
+  from {
+    transform: translateX(2rem);
+  }
+  to {
+    transform: translateX(0);
+  }
+}
+
+@keyframes fadeout {
+  from {
+    transform: translateX(2rem);
+  }
+  to {
+    transform: translateX(0);
+  }
+}
+
+.dialog {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  display: block;
+  border: 0;
+  padding: 0;
+  opacity: 1;
+  transition: opacity 0.2s;
+  background: rgba(0, 0, 0, 0.3);
+}
+
+dialog:not([open]) {
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: block;
+  height: 0;
+  opacity: 0;
+}
+
+.dialog__wrapper {
+  margin: 2em;
+  position: relative;
+}
+
+.dialog__document {
+  border: 1px solid #ccc;
```

Comment:
This border colour should match those used in our other borders `$color-borders`